ospf6d/ospf6_route.c

```diff
@@ -494,6 +493,25 @@ int ospf6_route_cmp(struct ospf6_route *ra, struct ospf6_route *rb)
 	return 0;
 }
 
+struct ospf6_route *ospf6_route_lookup_by_prefix(struct prefix *prefix)
```

Comment:
This function does not looks correct to me. This will return first prefix found in any of the ospf6 processes. It is possible that same prefix is present in multiple ospf6 due to either redistribution or other reasons. Can you please check it.

Reply:
This function is not required anymore.